crates/worker/src/partition_processor_manager.rs

```diff
@@ -648,6 +652,14 @@ impl<T: TransportConnect> PartitionProcessorManager<T> {
                 error!(%partition_id, error=%err, "Starting partition processor failed");
                 self.running_partition_processors.remove(&partition_id);
             }
+            ProcessorEvent::Stopped(err) => {
+                error!(%partition_id, error=%err, "Partition processor exited");
```

Comment:
Maybe `warn` because the system is not yet broken (hopefully). `warn!(%partition_id, "Partition processor exited unexpectedly: {err}");`